jl4-core/src/L4/EvaluateLazy.hs

```diff
@@ -166,7 +166,7 @@ nfDirective (MkEvalDirective r expr env) = do
 
 data EvalDirectiveResult =
   MkEvalDirectiveResult
-    { range  :: !SrcRange -- ^ of the (L)EVAL / CONTRACT directive
+    { range  :: !(Maybe SrcRange) -- ^ of the (L)EVAL / CONTRACT directive
```

Comment:
doesn't need to be strict anymore as it doesn't make the SrcRange strict.

Reply:
Thanks for pointing this out!